src/main/resources/schemas/schemaFileSchema.json

```diff
@@ -72,21 +76,33 @@
         },
         "appBuildId": {
           "$ref": "#/definitions/stringArray"
+        },
+        "os": {
+          "$ref": "#/definitions/stringArray"
+        },
+        "experimentId": {
+          "$ref": "#/definitions/stringArray"
         }
       },
       "additionalProperties": false
     },
     "eventGroups": {
       "type": "array",
+      "minItems": 1,
       "items": {
         "title": "Event group",
         "type": "object",
```

Comment:
Thoughts on adding `"additionalProperties": false` to this object to prevent other kinds of drift?